_projects_lfe/dynawo.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Dynawo"
+show-title: false
+layout: project
+excerpt: "Dynaωo is an hybrid C++/Modelica open source suite of simulation tools for power systems. It aims at providing power system stakeholders with a transparent, flexible, interoperable and robust suite of simulation tools that could ease collaboration and cooperation in the power system community."
```

Comment:
limit the excerpt to 200 characters, else it will not show entirely on the home page : 
![image](https://github.com/user-attachments/assets/280c6928-c535-4cb5-8f4e-3b91a7b4dd80)